app/templates/editor.html

```diff
@@ -176,6 +176,10 @@ <h5 class="modal-title" id="channelLabel">Channel</h5>
     /* Create an array with the values of all the text boxes in a column */
     $.fn.dataTable.ext.order['dom-text'] = function (settings, col) {
         return this.api().column(col, { order: 'index' }).nodes().map(function (td, i) {
+            search: {
```

Comment:
Set the search mode inside the map function does not work on my side.
I think it should be set inside the DataTable constructor.

Reply:
Good catch:) I overllooked this part. It has to be the same as https://github.com/Chris230291/STB-Proxy/pull/68 just adapted to your code.